service/src/main/java/org/cbioportal/genome_nexus/service/internal/VEPEnrichmentService.java

```diff
@@ -57,20 +56,32 @@
     public void enrichAnnotation(VariantAnnotation variantAnnotation) {
         // modify JSON returned by VEP
         if (enrichers != null) {
-            Iterator<Entry<String, AnnotationEnricher>> it = enrichers.entrySet().iterator();
-            while (it.hasNext()) {
-                Entry<String, AnnotationEnricher> enricher = it.next();
```

Comment:
`id` is now part of the `AnnotationEnricher` interface, this simplifies the iteration.